packages/types/src/extensions/pn-any-stream-reference-based-types.ts

```diff
@@ -0,0 +1,12 @@
+import { PnReferenceBased } from '../extension-types';
+export {
+  ACTION,
+  IAddPaymentAddressParameters,
+  IAddRefundAddressParameters,
+} from './pn-any-reference-based-types';
+
+/** Parameters for the creation action */
+export interface ICreationParameters extends PnReferenceBased.ICreationParameters {
+  expectedFlowRate: string;
+  expectedStartDate: string;
```

Comment:
document the expected format of these parameters

Reply:
it's documented in the specs branch made by Yoann.